metadata/aquatics-2021-05-01-BTW.yml

```diff
@@ -85,17 +85,17 @@ metadata:
 #model, this would be the number of climate inputs (temperature, precip, solar
 #radiation, etc.).
     drivers:
-      status: present #options: absent, present, data_driven, propagates, assimilates
+      status: propogates #options: absent, present, data_driven, propagates, assimilates
```

Comment:
This is changed to propagates because the model incorporates driver uncertainty (the ensembles) into the model predictions of stream temperature